### DISCUSSION TITLE
Status Detection Fails for Bot Owner

### DISCUSSION THREAD

**Discussion by cmarley3-14** [Pycord-Development/pycord#898] [Q&A] [answered]
When trying to get the custom status (custom activity) of users, my bot successfully registers all the activities. However it refuses to detect mine, even when I have an actual activity, like playing a game. It's only when I exit the game, or change my status during the game, that the game is registered and printed, but my custom status is never detected. Any insights on why this might be happening? Here's the minimal code required to replicate the issue.

```py
import discord
from discord.ext import commands

_intents = discord.Intents.all()
bot = commands.Bot(command_prefix = "..", intents = _intents)

@bot.event
async def on_ready():
    print("hi")

@bot.event
async def on_presence_update(before:discord.Member, after:discord.Member):
    print(before.activity)
    print(after.activity)
    print(after.activities)

bot.run("TOKEN")
```

**Comment by Dorukyum** [comment]
This is probably related to #891.

**Reply by cmarley3-14** [reply]
I'll also add additional information:
This is the before/after pair when I have a status and game at the same time, and change my status:
```py
()
(<Game name='Visual Studio Code'>,)
```
Here is an example of a user coming online and their status being registered:
```py
()
(<CustomActivity name="Hm when you know but you don't want" emoji=<PartialEmoji animated=False name='😔' id=None>>,)
```
Here is likely when a user went from online to idle, so the status and activities stayed the same:
```py
(<CustomActivity name='I has VeRy Good GraMarSSSSS' emoji=None>, <Activity type=<ActivityType.playing: 0> name='ROBLOX' url=None details=None application_id=363445589247131668 session_id=None emoji=None>)
(<CustomActivity name='I has VeRy Good GraMarSSSSS' emoji=None>, <Activity type=<ActivityType.playing: 0> name='ROBLOX' url=None details=None application_id=363445589247131668 session_id=None emoji=None>)
```

**Comment by Dorukyum** [comment, marked as answer]
Fixed in #1365.
